packages/fiori/src/i18n/messagebundle.properties

```diff
@@ -98,6 +98,12 @@ NOTIFICATION_LIST_GROUP_EXPANDED=Expanded
 #XACT: ARIA announcement for timeline label
 TIMELINE_ARIA_LABEL=Timeline
 
+#XTXT: ARIA announcement for the TimelineItem status
```

Comment:
#XACT is more accurate in this case as this isn't a general text but rather an accessible predefined description.